libs/builder/templates/parts/member-remarks.html.nunj

```diff
@@ -0,0 +1,8 @@
+{%- set remarks = docNode | getJsDocTags('remarks') -%}
+
+{% if remarks %}
```

Comment:
should be `remarks.length` otherwise all members have the `Notes` title in the description because `remarks` is empty array

Reply:
good catch! pushed fix